src/packages/@ncigdc/modern_components/CohortComparison/CohortComparison.js

```diff
@@ -176,9 +176,7 @@ export default compose(
             palette={[set1_colour, set2_colour]}
             style={{ flex: 1, width: 100 }}
           />
-          <div
-            style={{ marginLeft: 50, flex: 1, width: 100, textAlign: 'center' }}
-          >
+          <div style={{ marginLeft: 50, flex: 1, width: 100 }}>
```

Comment:
the link should probably still be centered
![image](https://user-images.githubusercontent.com/3953093/30562088-4cebd4b4-9c8b-11e7-8bb9-bf16ebf11b18.png)

Reply:
how about this instead:

![image](https://user-images.githubusercontent.com/2880089/30563033-7a2e4a30-9c8e-11e7-83df-787768e6d246.png)

Reply:
works for me 👍